docs/guides/integrating-the-feature-hub.md

```diff
@@ -144,6 +144,33 @@ Services][feature-service-dependencies] that are [provided by Feature
 Apps][own-feature-service-definitions] which are loaded from a remote location,
 instead of being provided by the integrator.
 
+## Choosing an Integrator Technology
+
+The Feature Hub allows for different technology choices for the integrator as
```

Comment:
The Feature Hub provides different technology options for both the integrator and feature apps supported by the integrator.